src/ManualMode.ino

```diff
@@ -0,0 +1,51 @@
+
+#include <Smartcar.h>
+
+const int fSpeed = 70; //70% of the full speed forward
+const int bSpeed = -70; //70% of the full speed backward
+const int lDegrees = -75; //degrees to turn left
+const int rDegrees = 75; //degrees to turn right
+
+BrushedMotor leftMotor(8, 10, 9);
+BrushedMotor rightMotor(12, 13, 11);
+DifferentialControl control(leftMotor, rightMotor);
+
+SimpleCar car(control);
+
+void setup() {
+  
+  Serial3.begin(9600);
+  pinMode(14,INPUT);
+  pinMode(15,OUTPUT);
+}
+
+void loop() {
+  
+    char input = Serial3.read();
+
+    if (input == '0')
```

Comment:
Would be nice to have some comments here so it's clear what each statement is supposed to do! However you can see them in the variables you've set so it's not entirely necessary.